### PR TITLE
Update outdated comments and do some minor reworks

### PR DESCRIPTION
**Pull request by liubin**
Some comments are old or copy miss, not suitable for the code.

And the API doc for virtcontainers is old.

Fixes: #2187

### CODE REVIEW

**Review by jodh-intel — APPROVED** [review]
Nice! Thanks @liubin !

lgtm

**Comment by jodh-intel** [comment]
Could you change the title on the PR though? :smile:

File: src/runtime/virtcontainers/sandbox.go [hunk 1]
```diff
@@ -57,6 +57,10 @@ const (
 	DirMode = os.FileMode(0750) | os.ModeDir
 )
 
+var (
```
Comment (egernst) [inline src/runtime/virtcontainers/sandbox.go:60]: nice!

**Review by egernst — APPROVED** [review]
LGTM -- thanks! 

Curious, did you use a tool to identify some of this, or was this all your eyes?

**Comment by liubin** [comment]
> LGTM -- thanks!
> 
> Curious, did you use a tool to identify some of this, or was this all your eyes?

I'm not using any tools, just find these when learning code.

**Review by amshinde — APPROVED** [review]

**Comment by fidencio** [comment]
Howdy!

I'd like to ask you to rebase your PR atop of current `main`, as https://github.com/kata-containers/kata-containers/pull/2197 has been merged and it has a fix that unblocks the CI.

Please, do the rebase and feel free `@fidencio` in case you do not have the powers to run `/test`.

Sorry for the inconvenience and happy hacking!

**Comment by fidencio** [comment]
Hey, we've had yet another breakage in our CI and now it's fixed as part of https://github.com/kata-containers/kata-containers/pull/2203 ... it means that you'll need to rebase your PR atop of current main.

Sorry for the inconvenience.

**Pull request merged**